cmake/modules/grpc.cmake

```diff
@@ -125,6 +125,9 @@ else()
 				-DZLIB_ROOT:STRING=${ZLIB_SRC}
 			BUILD_IN_SOURCE 1
 			BUILD_BYPRODUCTS ${GRPC_LIB} ${GRPCPP_LIB} ${GPR_LIB} ${GRPC_LIBRARIES}
+			# Keep installation files into the local ${GRPC_INSTALL_DIR} 
+			# since here is the case when we are embedding gRPC
+			INSTALL_COMMAND DESTDIR= ${CMD_MAKE} install
```

Comment:
N.B. for reviewers:
the resulting installation directory is computed by `DESTDIR` + `CMAKE_INSTALL_PREFIX`.
Basically, it forces gRPC installation files to be always installed into the path specified by `CMAKE_INSTALL_PREFIX` that's set to `${GRPC_SRC}/target` (see [here](https://github.com/falcosecurity/libs/blob/master/cmake/modules/grpc.cmake#L94)).

See also:
- https://cmake.org/cmake/help/latest/envvar/DESTDIR.html
https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html